### PR TITLE
Main User Page Created

### LINKED ISSUE #93: create Main User Page
**Is your feature request related to a problem? Please describe.**
Right now admin portal doesn't have the main user page, so I want to work on that.

**Describe the solution you'd like**
Work on the main page as per the Figma design

**Additional details**
GitHub Externship : 19-05_Sin666_tfa_262

Please assign me this task @DangaRanga sir, @palisadoes sir

### PR DESCRIPTION
**Pull request by ssaumyaa7**
**What kind of change does this PR introduce?**
Created Main User Page

**Issue Number:**
Fixes #93 

**Did you add tests for your changes?**
Yes

**Summary**
Added page to show information about the users, this 

**Additional details**
GitHub Externship : 19-05_Sin666_tfa_262

**Mentor**
@DangaRanga sir, please merge

### CODE REVIEW

File: src/components/AdminNavbar/AdminNavbar.module.css [hunk 1]
```diff
@@ -64,13 +64,13 @@ a {
 
 .sidebar {
   z-index: 0;
-  position: relative;
+  padding-top: 50px;
+  position: fixed;
```
Comment (DangaRanga) [inline src/components/AdminNavbar/AdminNavbar.module.css:68]: These changes had resulted in the organizations not being displayed. So please revert them. 
image

File: src/screens/MainUserPage/MainUserPage.module.css [hunk 2]
```diff
@@ -0,0 +1,327 @@
+.wrappage {
```
Comment (DangaRanga) [inline src/screens/MainUserPage/MainUserPage.module.css:1]: Although this is an extreme case, please apply a flex-direction: column, to the interests segment, when the device is below a certain width/media breakpoint, as the interest bubbles overflow horizontally as the page is resized

image

File: src/screens/MainUserPage/MainUserPage.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
+import React from 'react';
```
Comment (DangaRanga) [inline src/screens/MainUserPage/MainUserPage.tsx:1]: For this component, you could reuse the SuperOrgHome component as opposed to rebuilding an entire component from scratch
image

**Review by DangaRanga — CHANGES_REQUESTED** [review]

File: src/screens/MainUserPage/MainUserPage.tsx (hunk 3 above)
Comment (ssaumyaa7) [reply src/screens/MainUserPage/MainUserPage.tsx:1]: Sir I have not created this specific page.

File: src/components/AdminNavbar/AdminNavbar.module.css (hunk 1 above)
Comment (ssaumyaa7) [reply src/components/AdminNavbar/AdminNavbar.module.css:68]: I guess this is Admin Navbar page so nothing is to be displayed over here.

File: src/screens/MainUserPage/MainUserPage.module.css (hunk 2 above)
Comment (ssaumyaa7) [reply src/screens/MainUserPage/MainUserPage.module.css:1]: Will do that.

**Review by DangaRanga — CHANGES_REQUESTED** [review]

**Pull request closed without merging**